Live/Classes/MainScene.cpp

```diff
@@ -22,8 +23,15 @@ bool MainScene::init()
     m_player = new Player("Spritesheet/roguelikeChar_transparent.png",
             SPRITE_GRID_X, SPRITE_GRID_Y, 100,100);
     // extract the m_player from the m_playersheet
-    this->addChild(m_player->getSprite(), 0);
+    this->addChild(m_player->getSprite(), INT_MAX); // Player always on top
```

Comment:
hmm this is not true actually, but making this assumption is okay for now. Just make sure to update this when adding height mapping of trees/tall structures later on.